### PR TITLE
Proof Read several Milvus Lite integrations and convert to notebook

### PR DESCRIPTION
**Pull request by Jacksonxhx**
- [ ] A reference to a related issue in your repository.
  
  Each PR is related to an issue, and you need to list that issue.

- [ ] A description of the changes proposed in the pull request.

  A brief introduction to this PR.

- [ ] Add delight to the experience when all tasks are complete :tada:

### CODE REVIEW

**Comment by Jacksonxhx** [comment]
1. proof read several integrations and append them in tutorial integrations
2. add quickstart in tutorial quickstart

File: bootcamp/RAG/readthedocs_zilliz_langchain.ipynb [hunk 1]
```diff
@@ -913,8 +913,8 @@
     "# Define a sample question about your data.\n",
     "QUESTION1 = \"What do the parameters for HNSW mean?\"\n",
     "QUESTION2 = \"What are good default values for HNSW parameters with 25K vectors dim 1024?\"\n",
-    "QUESTION3 = \"What is the default AUTOINDEX distance metric in Milvus Client?\"\n",
-    "QUESTION4 = \"What does nlist mean in ivf_flat?\"\n",
+    "QUESTION3 = \"What does nlist vs nprobe mean in ivf_flat?\"\n",
```
Comment (codingjaguar) [inline bootcamp/RAG/readthedocs_zilliz_langchain.ipynb:916]: Why changing these two lines?

**Review by codingjaguar — COMMENTED** [review]
Please change JINA to Jina everywhere.

File: bootcamp/tutorials/integration/milvus_and_DSPy.ipynb [hunk 2]
```diff
@@ -0,0 +1,873 @@
+{
```
Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: <p>Can you zoom out the picture a bit? It's taking too much space.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !pip install pymilvus==2.4.2
</pre><p>!pip install -U pymilvus</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: <p>Hide the output for this section. It's not important.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: <h3>Ingest data into the Milvus vector database </h3><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: <p>Clear the output for this section.</p><p> </p><p>BTW the output shows the code doesn't work. Can you double check?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

File: bootcamp/tutorials/integration/milvus_with_JINA.ipynb [hunk 3]
```diff
@@ -0,0 +1,1205 @@
+{
```
Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_with_JINA.ipynb:1]: <p>clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/#comment-1609395057'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (codingjaguar) [inline bootcamp/tutorials/integration/milvus_with_JINA.ipynb:1]: <p>this failed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/#comment-1609395061'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: bootcamp/tutorials/integration/semantic_search_with_milvus_and_voyageai.ipynb [hunk 4]
```diff
@@ -0,0 +1,191 @@
+{
```
Comment (codingjaguar) [inline bootcamp/tutorials/integration/semantic_search_with_milvus_and_voyageai.ipynb:1]: <p>clear the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/#comment-1609395066'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: bootcamp/tutorials/quickstart/quickstart.ipynb [hunk 5]
```diff
@@ -0,0 +1,601 @@
+{
```
Comment (codingjaguar) [inline bootcamp/tutorials/quickstart/quickstart.ipynb:1]: <h1>Quickstart with Milvus Lite</h1><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/#comment-1609395072'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (codingjaguar) [inline bootcamp/tutorials/quickstart/quickstart.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.    # data: ["[{'id': 2, 'distance': 0.5859944820404053, 'entity': {'text': 'Born in Maida Vale, London, Turing was raised in southern England.', 'subject': 'history'}}, {'id': 1, 'distance': 0.5118255019187927, 'entity': {'text': 'Alan Turing was the first person to conduct substantial research in AI.', 'subject': 'history'}}]"] , extra_info: {'cost': 0}
</pre><p>No need to add this kind of comment in notebook. Just show the result in output cell. Please fix everywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

File: bootcamp/workshops/dbta_may_2024/1. RAG_basic.ipynb [hunk 6]
```diff
@@ -28,7 +28,7 @@
   },
```
Comment (codingjaguar) [inline bootcamp/workshops/dbta_may_2024/1. RAG_basic.ipynb:28]: <p>Why changing this file at all?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

File: bootcamp/RAG/readthedocs_zilliz_langchain.ipynb (hunk 1 above)
Comment (Jacksonxhx) [reply bootcamp/RAG/readthedocs_zilliz_langchain.ipynb:916]: This is the commit by a previous person

File: bootcamp/tutorials/integration/milvus_and_DSPy.ipynb (hunk 2 above)
Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: solved

Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: solved

Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: solved

Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: solved

Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_and_DSPy.ipynb:1]: solved

File: bootcamp/tutorials/integration/milvus_with_JINA.ipynb (hunk 3 above)
Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_with_JINA.ipynb:1]: solved

Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/milvus_with_JINA.ipynb:1]: solved

File: bootcamp/tutorials/quickstart/quickstart.ipynb (hunk 5 above)
Comment (Jacksonxhx) [reply bootcamp/tutorials/quickstart/quickstart.ipynb:1]: solved

Comment (Jacksonxhx) [reply bootcamp/tutorials/quickstart/quickstart.ipynb:1]: solved

File: bootcamp/workshops/dbta_may_2024/1. RAG_basic.ipynb (hunk 6 above)
Comment (Jacksonxhx) [reply bootcamp/workshops/dbta_may_2024/1. RAG_basic.ipynb:28]: This is a commit by a previous person

**Comment by Jacksonxhx** [comment]
A new version has been updated

File: bootcamp/tutorials/integration/semantic_search_with_milvus_and_voyageai.ipynb (hunk 4 above)
Comment (Jacksonxhx) [reply bootcamp/tutorials/integration/semantic_search_with_milvus_and_voyageai.ipynb:1]: <p>solved</p>

**Pull request merged**